lisa/transformers/kernel_source_installer.py

```diff
@@ -143,7 +146,7 @@ def install(self) -> str:
         runbook: SourceInstallerSchema = self.runbook
         assert runbook.location, "the repo must be defined."
 
-        self._install_build_tools(node)
+        self._install_build_tools(node, runbook.build_deps)
```

Comment:
Can the build_deps be added into this class, instead of configurable? It makes repro harder from others.

Reply:
Can you please elaborate?

Currently it is added as a configurable because different kernel versions might need different build dependencies.

Reply:
You can add checks for kernel versions, if it's some version, add the deps. If the deps are maintained out of code, it's hard to others to make the build tools works without the right configuration.

Reply:
Yeah, but we fetch the kernel version after installation of the build tools (otherwise `make kernelrelease` might fail). Should I add a second call to install_build_tools after kernel version is known?